aws/_docs/rds-plan.md

```diff
@@ -0,0 +1,30 @@
+# RDS Plan
+
+Hello!
+
+Looking up to the existing tests it's possible to create something similar that creates and deletes the instances so I'll try to avoid using the UI. I'm thinking about following these steps:
+
+1. Create a new RDS instance
+1. The lookp for the instances based on region
+1. Nuke the instance based on the lookup results (with SkipFinalSnapshot: true)
```

Comment:
`SkipFinalSnapshot: true` is the right approach for now. As one possible option for the future, this should be configurable (e.g., via a command line param such as `--rds-skip-final-snapshot`) to let the user choose. But don't bother with that until the basics work.